### ISSUE TITLE
제안 사항

### ISSUE THREAD

**Issue by jeeseongmin** [jeeseongmin/oss_miniproject#10]
지금 주제를 제대로 정하지 못한 상황에서 뼈대를 올리는 것보다 다 지우고 처음부터 해보는 게 어떨까요?
주제는 <학생 성적 관리> 제안해봅니다.

**Comment by ykeab94** [comment]
네! 좋아요~ ^^

**Comment by jeeseongmin** [comment]
그러면 학우님께서 기본적인 CRUD를 맡아주시겠어요? 
저는 기초적인 main 함수와 파일 관련 함수(저장, 불러오기) 구축을 맡겠습니다.

**Comment by ykeab94** [comment]
네! 그럼 제가 CRUD에서 필요한 학생의 점수 추가, 점수 수정, 점수 삭제, 번호 선택(selectNo) 함수들을 추가하겠습니다!

**Comment by jeeseongmin** [comment]
제가 먼저 파일들 구성하겠습니다. 잠시만 기다려주세요

**Comment by ykeab94** [comment]
그 학생들 성적 입력란을 이렇게 구분하면 어떨까요?
그 세부 구성은
-------------------------
이름
중간 (40점만점)
lab (20점만점 - lab 20개 있다고 치고 한개당 1점) 
프로젝트 (10점)
기말 (30점만점)
--------------------------
이렇게요!!

**Comment by jeeseongmin** [comment]
오 좋습니다!!
참고해서 wiki에 넣고 파일 곧 올릴께요!
